src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventsSink.scala

```diff
@@ -0,0 +1,102 @@
+package com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra
+
+import java.util.UUID
+
+import com.datastax.spark.connector.writer.WriteConf
+import com.microsoft.partnercatalyst.fortis.spark.dto.FortisEvent
+import org.apache.spark.sql.{Dataset, SaveMode, SparkSession}
+import org.apache.spark.streaming.dstream.DStream
+import com.datastax.spark.connector._
+import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.aggregators._
+import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.dto._
+import org.apache.spark.rdd.RDD
+import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.udfs._
+import org.apache.spark.streaming.StreamingContext
+
+object CassandraEventsSink {
+  private val KeyspaceName = "fortis"
+  private val TableEvent = "events"
+  private val TableEventTags = "eventtags"
+  private val TableEventBatches = "eventbatches"
+  private val CassandraFormat = "org.apache.spark.sql.cassandra"
+
+  def apply(dstream: Option[DStream[FortisEvent]], ssc: StreamingContext): Unit = {
+    if (dstream.isDefined) {
+
+      val aggregators = Seq(new PopularPlacesAggregator)
+      val batchStream = dstream.get.foreachRDD(eventsRDD => {
+        eventsRDD.foreachPartition(partition => {
```

Comment:
This was a mistake on my end as I forgot to remove the `forEachPartition` call from an alternative approach I attempted earlier. I'll remove.